evm_loader/program/src/executor/state.rs

```diff
@@ -449,20 +453,39 @@ impl<'a, B: AccountStorage> Database for ExecutorState<'a, B> {
             .collect::<Vec<_>>();
 
         if !metas.iter().any(|m| (m.pubkey == address) && m.is_writable) {
+            // account with given address don't changed by current actions
+            // put offset to cache
```

Comment:
Please use a spell checker when writing comments in English.
Also, in general, comments should explain why something is happening, not what.